<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="com.webappsecurity.zero.testscripts.VerifyValidLoginTest">
        <methods>
          <include name="verifyValidLogin"/>
          <include name="openapplication"/>
          <include name="closeapplication"/>
        </methods>
      </class> <!-- com.webappsecurity.zero.testscripts.VerifyValidLoginTest -->
      <class name="com.webappsecurity.zero.testscripts.VerifyFundTransferTest">
        <methods>
          <include name="doFundTransfer"/>
          <include name="openapplication"/>
          <include name="closeapplication"/>
        </methods>
      </class> <!-- com.webappsecurity.zero.testscripts.VerifyFundTransferTest -->
      <class name="com.webappsecurity.zero.testscripts.VerifyInvalidLoginTest">
        <methods>
          <include name="InvalidLogin"/>
          <include name="openapplication"/>
          <include name="closeapplication"/>
        </methods>
      </class> <!-- com.webappsecurity.zero.testscripts.VerifyInvalidLoginTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
